backend/notifications/views.py

```
@@ -57,3 +63,39 @@ def mark_read(self, request, pk=None):
         return Response({'updated': updated})
+
+
+class WhatsNewPagination(PageNumberPagination):
+    page_size = 20
+    page_size_query_param = 'page_size'
+    max_page_size = 50
+
+
+class WhatsNewAnnouncementViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+    serializer_class = WhatsNewAnnouncementSerializer
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)